### ISSUE TITLE
Apakah kita perlu membuat desain di figma?

### ISSUE THREAD

**Issue by ginanjar10119032** [m10119002/SIM-PaudMentari#4]
Karena kita kan sudah punya gambaran desain yang cukup jelas di google drive

**Comment by m10119002** [comment]
gambaran kebut semalam

**Comment by m10119002** [comment]
figma boleh, mesti konsul dulu ke @agdd1709

**Comment by ginanjar10119032** [comment]
betul, tapi untuk sekarang lebih baik fokus ke mempercantik homepage web kita. supaya kejadian yang menimpa kelompok ayam tidak kita alami.

**Comment by ginanjar10119032** [comment]
stuck di desain

**Comment by m10119002** [comment]
desain profil sekolah gk terlalu ribet kali klo dibanding sama kelompok ayam, demografiknya kan orang tua.. ya paling foto2 fasilitas/sarana, profil guru, foto murid lagi main di halaman, foto murid lagi belajar, dll.

kan fokusnya ke desain HP, jadi model titik gitu, jadi misal ada 3 pengumuman.. nanti di kotak pengumuman ada tiga titik dibawahnya bisa di geser ke kanan-kiri

**Comment by m10119002** [comment]
ya itu sih, masalah selanjutnya tampilan mau pakai framework atau dari awal aja buatnya

**Comment by ginanjar10119032** [comment]
untuk pemakaian jangka lama mending pake framework biar lebih teratur

**Comment by ginanjar10119032** [comment]
tapi aku ga yakin bisa bikin yg model 3 titik gitu. walaupun pake framework

**Comment by m10119002** [comment]
lagi nyari "front end framework html5" di google, sekarang lagi cek purecss.io
research aja dulu fitur yang ditawarkan framework apa aja, aku lagi nyari yang simpel tapi pasti kepake gitu
